src/web/src/components/Banner.tsx

```diff
@@ -87,6 +87,22 @@ const useStyles = makeStyles((theme: Theme) =>
         display: 'grid',
       },
     },
+    planet: {
+      position: 'absolute',
+      bottom: theme.spacing(3),
+      right: theme.spacing(3),
+      color: 'white',
+    },
+    planetAnchor: {
+      opacity: 0.85,
+      fontFamily: 'Spartan',
+      fontSize: '1.3em',
```

Comment:
I feel like the font size needs to be increased just a little bit. It's a little hard for me to view. Also, the opacity makes the text hard to view depending on certain images the image service generates.
I wanna know how others are viewing this though.

![image](https://user-images.githubusercontent.com/16841702/150065452-26d6df9e-6473-4939-b07b-bb4c694b4d89.png)

Reply:
I agree that the font size could use a little increase. Indifferent about the opacity.

Reply:
I think it's the same font size as the version number @ `1.3em`

Reply:
@HyperTHD Let's try with this font size and, if we're not happy, we can change it in the future.